Program.cs

```diff
@@ -3,6 +3,10 @@
 using FFMpegCore;
 using FFMpegCore.Helpers;
 using QuickEdit.Commands;
+using QuickEdit.Logger;
+using QuickEdit.LoggerConfig;
```

Comment:
If `AutoLog` is moved to the `QuickEdit.Logger` namespace, the `QuickEdit.LoggerConfig` using can be removed
```suggestion
```